recipes/austin/bld.bat

```diff
@@ -0,0 +1,6 @@
+autoreconf --install
+./configure --prefix=%PREFIX%
+make
+make install
+
+%PYTHON% -m pip install . -vv
```

Comment:
This file should have exit conditions after each line

Reply:
Ok, updated. Let me know if it's not what you were looking for.

Reply:
Yep this is correct